src/pages/Scholarships.jsx

```diff
@@ -41,6 +43,9 @@ function ScholarshipsPage() {
         (loading && <CircularProgress className={classes.progress} />) || (
           <ScholarshipList scholarships={scholarships} />
         )}
+      <Divider />
+
+      <Pagination count={10} />
```

Comment:
The [Figma design](https://www.figma.com/file/nwrW3UebcvSWyQZGb5ixNr/DreamScholar?node-id=0%3A1) has theming on it. You can achieve using the theme's primary color adding `color="primary"`. Also, I think the size should be [Large](https://material-ui.com/components/pagination/#pagination-size).

Reply:
Instead of this Pagination component, since we can't know the number of pages beforehand, let's just have a simple **Load More Results** button that loads more results when clicked on it. You don't have to get it working as part of this PR, you can get it to work as part of addressing #94. Eventually we may replace the button with auto-loading when the user scrolls to the bottom but to keep things simple let's just add a simple button.